packages/protocol-devtools-solana/src/endpointv2/sdk.ts

```diff
@@ -548,6 +562,11 @@ export class EndpointV2 extends OmniSDK implements IEndpointV2 {
         throw new TypeError(`isRegisteredLibrary() not implemented on Solana Endpoint SDK`)
     }
 
+    @AsyncRetriable()
+    async isBlockedLibrary(): Promise<boolean> {
```

Comment:
@ziming-zung is this fine to include? it otherwise wouldn't build, complaining about how IEndpointV2 needs `isBlockedLibrary`